runtime/doc/helphelp.txt

```diff
@@ -315,7 +315,7 @@ Hints for translators:
   using the "tag@en" notation.
 - Make a package with all the files and the tags file available for download.
   Users can drop it in one of the "doc" directories and start use it.
-  Report this to Bram, so that he can add a link on www.vim.org.
+  Report to the development team, so they can add a link on www.vim.org.
```

Comment:
it seems called `the vim project team`.
// and tho the info is out-of-date, but somewhere kept the information or markers or name of Bram it ok.
// e.g `the vim project team, successor of Bram.`